ui/admin/app/styles/app.scss

```diff
@@ -740,7 +740,8 @@
 }
 
 .auth-methods-list-table,
-.workers-list-table {
+.workers-list-table,
+.sessions-list-table {
```

Comment:
Do we want to create a more generic class name for this and spread it to all of the tables that need?

Reply:
In previous HDS work I left this comment: "If this specific change is causing friction or raising doubts, I will suggest to stick the PR to rafctor the HDS::Table and bring the toolbar issue to tech time. Toolbar is used in different places of our UI, so can be good to dedicate a specific ticket/PR to "normalize" the toolbar."

My opinion here is: (feel free to discuss or NOT agree)
- We have lots of tables.
- We have some of those tables also using a toolbar component.
- Let's work and finish first the tables. That will give us a better understanding of the toolbar "issue".
- Once we have better understanding, we can tackle it in a specific task and PR where the topic is just the toolbar.

Reply:
@calcaide @cameronperera thank you both for the information!  Based on this conversation and the one from tech-time, I will just use the custom CSS.